src/frontend/src/common/redux/middleware/index.ts

```diff
@@ -56,11 +61,25 @@ export const setGroupMiddleware: Middleware =
     return next(action);
   };
 
+const expirePathogenCaches = async (): Promise<void> => {
```

Comment:
Is this the right file to put this in?  The groups one is in `src/common/queries/groups.ts`.  This one has query keys from multiple queries files, so I put it here.  Maybe we would want a separate queries file for it?

Reply:
I wasn't part of the original conversation around this, but it seems like a reasonable place to me, at least until we have to start wrangling more of these. This is the only file `expirePathogenCaches` gets called right now, so having it next to where it gets called seems as good as anything else.